docs/cloud/concepts/projects.md

```diff
@@ -24,7 +24,20 @@ query {
 }
 ```
 
-## Creating a new project <Badge text="GQL"/>
+## Creating a new project
```

Comment:
There's also a CLI command for this! `prefect create project`

Reply:
Oh nice! I’ll update to include that

Reply:
Updated